src/views/authenticated/ManageList.tsx

```diff
@@ -154,6 +163,8 @@ export function ManageList({ listPath }: Props) {
 							Submit Item
 						</button>
 					</form>
+					<h4>Let&apos;s look at your list!</h4>
+					<button onClick={navigateToListPage}>{"Let's go!"}</button>
```

Comment:
praise: I like adding a button to get back to the list!

Reply:
The button to get back to the list is definitely a great added feature that improves user experience and functionality. I was thinking, from our  group conversation last night about accessibility, it would be a great idea to rename the button perhaps to "View List" instead of "Let's go!"! so it speaks more to the buttons purpose :) and then perhaps the text above it "Let's look at your list" would be optional? I do like how it's so conversational though!